module/Company/Module.php

```diff
@@ -146,8 +166,10 @@ private function getOtherFactories()
                 $acl->allow('guest', 'company', 'list');
                 $acl->allow('guest', 'company', 'view');
                 $acl->allow('guest', 'company', 'listVisibleCategories');
+                $acl->allow('guest', 'company', 'listVisibleLabels');
```

Comment:
@Koen1999 Is this privilege still used now that you removed the hidden property?

Reply:
Yes it is. Invisible labels are defined as "labels that are not assigned to any active job" in this context